docs/mllib-linear-methods.md

```diff
@@ -78,6 +78,10 @@ methods `spark.mllib` supports:
   </tbody>
 </table>
 
+A binary label y is denoted as either +1 (positive) or −1 (negative), which is
```

Comment:
This duplicates an existing statement below. The idea was to move it u here rather than copy it.